cardano-cli/src/Cardano/CLI/Shelley/Output.hs

```diff
@@ -22,26 +22,36 @@ data QueryTipOutput = QueryTipOutput
   , syncProgress :: Maybe Text
   } deriving (Eq, Show)
 
+-- | A key-value pair difference list for encoding a JSON object.
+(..=) :: (KeyValue kv, ToJSON v) => Text -> v -> [kv] -> [kv]
```

Comment:
Interesting :smile: . What's the benefit of using a difference list here?

Reply:
Normally, I would say performance, but that's not needed in this case.

Here, mainly, I wanted to be able to toggle list membership element be element and syntactically, this was the most pleasant I could find (for the moment).